### ISSUE TITLE
TF tutorial getting started fix

### ISSUE THREAD

**Issue by Lee-L-Boyd** [tensorflow/tensorflow#10148]
Hi, 

I am not personally having an issue, but I wanted to make you aware of some things in the tutorial that might throw off a new user.  

When the tutorial starts talking about the high level functions, they define the epochs here:

input_fn = tf.contrib.learn.io.numpy_input_fn({"x":x}, y, batch_size=4, num_epochs=1000)

but then they redefine here:

estimator.fit(input_fn=input_fn, steps=1000)

This seems redundant, and per earlier issues:
https://github.com/tensorflow/tensorflow/issues/10042
https://github.com/tensorflow/tensorflow/issues/7677

There were people who said this made their system glitch.  If the system did accept both arguments, then I am not clear, nor have I found anywhere in the documentation, where it says which argument would take precedence if they are different numbers.

On another note, earlier in the tutorial, it where it calculates print(sess.run(linear_model, {x:[1,2,3,4]})), for some reason there is a floating decimal point way at the end of the output:

[ 0.          0.30000001  0.60000002  0.90000004]

This doesn't add up, unless there is just some kind of extra operation that is not being shown.  I know it is likely a lot of work to handle all the issues that you get, but fixing these things might reduce your workload over time, as new users will be less confused up front, not to mention keep users from getting discourages and changing to other platforms before they begin.  Thank you for your help.

**Comment by Lee-L-Boyd** [comment]
Hi again,
I see the difference between step and epoch now.  Step is the number of batches you need to complete an epoch apparently.  E.g. steps=totalSamples/batchSize.  Still makes very little sense to me in the tutorial, given batchSize==totalSamples, although the problem isn't terrible if it goes unresolved.  I still have no explanation for the numbers being off in the latter example?  Thanks again!

**Comment by twins14** [comment]
Hello Lee-L-Boyd, 

so i am twins14  and just a beginner, who is really intersting in learning tensorflow in my sparetime :))
But i really need some help :((
i am looking for a machine learning example like the "image classification" to reconstruct it.
So my question to you is if you have such a code example ? 👍 
When not, do you have maybe other little examples in Tensorflow for me to begin learning 💯 
My email adress is: almin-b@hotmail.de

So i would be very glad hearing from you :)

Wish you a nice evening!

**Comment by Lee-L-Boyd** [comment]
Hi twins14,

I am new to TF myself.  What I have been doing is going over the mnist
tutorial, which is basically just classifying numbers in photos:

https://www.tensorflow.org/get_started/mnist/beginners

with the actual code you can run here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist.py

I have moved on to the graph visualizations tutorial here:

https://www.tensorflow.org/get_started/summaries_and_tensorboard

and code for it here:

https://github.com/tensorflow/tensorflow/blob/r1.1/tensorflow/examples/tutorials/mnist/mnist_with_summaries.py

I think understanding the visualization portion will be very important if
you want to actually test your code.  Goodluck!


*Lee L. Boyd*
President  - CSGSA at UTSA, 2017
r <lee.l.boyd@gmail.com>cx653@my.utsa.edu
(210)-404-3607

**Comment by twins14** [comment]
Thank you very much for your support 👍

Mittwoch, 24 Mai 2017, 10:55nachm. +02:00 von Lee-L-Boyd notifications@github.com<mailto:notifications@github.com>:

Hi twins14,

I am new to TF myself. What I have been doing is going over the mnist
tutorial, which is basically just classifying numbers in photos:

https://www.tensorflow.org/get_started/mnist/beginners

with the actual code you can run here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist.py

I have moved on to the graph visualizations tutorial here:

https://www.tensorflow.org/get_started/summaries_and_tensorboard

and code for it here:

https://github.com/tensorflow/tensorflow/blob/r1.1/tensorflow/examples/tutorials/mnist/mnist_with_summaries.py

I think understanding the visualization portion will be very important if
you want to actually test your code. Goodluck!


*Lee L. Boyd*
President - CSGSA at UTSA, 2017
r <lee.l.boyd@gmail.com<mailto:lee.l.boyd@gmail.com>>cx653@my.utsa.edu
(210)-404-3607

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.
